### PR TITLE
Adding http sink connector example [skip secret scan]

### PR DESCRIPTION
**Pull request by MosheBlumbergX**
In this example we create http sink connector with validation steps

### CODE REVIEW

File: connector/http-sink-connector/producer-app-data.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kafka-client-config
+  namespace: confluent
+type: Opaque
+data:
+  kafka.properties: Ym9vdHN0cmFwLnNlcnZlcnM9a2Fma2EuY29uZmx1ZW50LnN2Yy5jbHVzdGVyLmxvY2FsOjkwNzEKc2VjdXJpdHkucHJvdG9jb2w9UExBSU5URVhU
```
Comment (domenicbove) [inline connector/http-sink-connector/producer-app-data.yaml:9]: id recommend doing
```suggestion
stringData:
  kafka.properties: |
    <put the decoded here>
```
clearer this way

File: connector/http-sink-connector/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+Once the Kafka cluster is ready create the secret, topic, and producer, all within one YAML file:    
+```
+kubectl --namespace confluent apply -f $TUTORIAL_HOME/producer-app-data.yaml
+```  
+
+Check topic 
+```
+kubectl --namespace confluent get topic
+```
```
Comment (domenicbove) [inline connector/http-sink-connector/README.md:70]: could also consume from the topic here

File: connector/http-sink-connector/README.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+Deploy Confluent for Kubernetes (CFK): 
+
+```
+helm upgrade --install confluent-operator \
+  confluentinc/confluent-for-kubernetes \
+  --namespace confluent
+```
+
+## Deploy nodejs API server
```
Comment (domenicbove) [inline connector/http-sink-connector/README.md:32]: maybe explain that the webserver is a custom built image and how it works. accepts simple PUT requests? and stores all messages on an endpoint

**Review by domenicbove — COMMENTED** [review]
Very cool! some minor comments, maybe add more to the description at the top, it wasn't clear at first read what each piece would do

File: connector/http-sink-connector/producer-app-data.yaml (hunk 1 above)
Comment (MosheBlumbergX) [reply connector/http-sink-connector/producer-app-data.yaml]: great idea!

File: connector/http-sink-connector/README.md (hunk 2 above)
Comment (MosheBlumbergX) [reply connector/http-sink-connector/README.md:70]: The consumer step is part of the validation, I hope that's OK ?

**Comment by MosheBlumbergX** [comment]
thanks @domenicbove , great ideas - thanks for the input. 
I've made changes and noted that the consumer is added later on in the validation steps

File: connector/http-sink-connector/producer-app-data.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+metadata:
+  name: kafka-client-config
+  namespace: confluent
+type: Opaque
+data:
+  kafka.properties: |
+    <put the decoded here>
+# for example   Ym9vdHN0cmFwLnNlcnZlcnM9a2Fma2EuY29uZmx1ZW50LnN2Yy5jbHVzdGVyLmxvY2FsOjkwNzEKc2VjdXJpdHkucHJvdG9jb2w9UExBSU5URVhU
+
+
```
Comment (domenicbove) [inline connector/http-sink-connector/producer-app-data.yaml:13]:
```suggestion
stringData:
  kafka.properties: |
    bootstrap.servers=kafka.confluent.svc.cluster.local:9071
    security.protocol=PLAINTEXT
```

**Comment by MosheBlumbergX** [comment]
@domenicbove , can you kindly check this one? thanks!

File: connector/http-sink-connector/producer-app-data.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
```
Comment (domenicbove) [inline connector/http-sink-connector/producer-app-data.yaml:1]: maybe lose this blank line?

**Review by domenicbove — APPROVED** [review]
LGTM! i created the secret and made sure it encoded/decoded correctly

**Pull request merged**